typescript/phoenix-sdk/examples/simpleMarketMaker.ts

```diff
@@ -149,90 +145,80 @@ export async function simpleMarketMaker(privateKeyPath: string) {
     // Get current time in seconds
     const currentTime = Math.floor(Date.now() / 1000);
 
-    // create bid and ask instructions for 1 SOL
-    const bidOrderPacket: Phoenix.OrderPacket = {
-      __kind: "PostOnly",
-      side: Phoenix.Side.Bid,
-      priceInTicks: Phoenix.toBN(
-        client.floatPriceToTicks(bidPrice, marketPubkey.toString())
-      ),
-      numBaseLots: Phoenix.toBN(
-        client.rawBaseUnitsToBaseLotsRoundedDown(1, marketPubkey.toString())
-      ),
-      clientOrderId: Phoenix.toBN(1),
-      rejectPostOnly: false,
+    // Create a LimitOrderTemplate for the bid and ask orders.
+    // the LimitOrderTemplate allows you to specify the price and size in commonly understood units:
+    // price is simply the floating point price as represented on most exchanges, and size is the number of base whole units (i.e. one SOL token).
```

Comment:
"price is the floating point price (units of USDC per unit of SOL for the SOL/USDC market)"
"size is in whole base units (units of SOL for the SOL/USDC market)"